src/main/groovy/wooga/gradle/release/ReleasePlugin.groovy

```diff
@@ -124,18 +128,23 @@ class ReleasePlugin implements Plugin<Project> {
 
         project.pluginManager.apply(VersionPlugin)
         VersionPluginExtension versionExtension = project.extensions.findByType(VersionPluginExtension)
-        def versionScheme = project.properties.getOrDefault("version.scheme", VERSION_SCHEME_DEFAULT)
-        versionExtension.with {
-            switch (versionScheme) {
-                case VERSION_SCHEME_SEMVER_2:
-                    versionExtension.versionScheme(VersionScheme.semver2)
-                    break
-                case VERSION_SCHEME_SEMVER_1:
-                default:
-                    versionExtension.versionScheme(VersionScheme.semver)
-                    break
-            }
-        }
+        // TODO: Do we need to map this here if the version plugin itself already watches this property (version.scheme)?
+        // TODO: Set only the convention to 'wdk' here?
+        //def versionScheme = project.properties.getOrDefault("version.scheme", VERSION_SCHEME_DEFAULT)
+//        versionExtension.with {
+//            switch (versionScheme) {
+//                case VERSION_SCHEME_SEMVER_2:
+//                    versionExtension.versionScheme(VersionScheme.semver2)
+//                    break
+//                case VERSION_SCHEME_SEMVER_1:
+//                    versionExtension.versionScheme(VersionScheme.semver)
+//                    break
+//                case VERSION_SCHEME_WDK:
+//                    versionExtension.versionScheme(VersionScheme.wdk)
+//                    break
+//            }
+//        }
+        versionExtension.versionScheme.convention(VersionScheme.wdk)
```

Comment:
No I mean adjusting the `VERSION_SCHEME_DEFAULT` to point to `wdk`

Reply:
Or wait. Scrap this. Leave the old way for now and we deal with this in the WDKs for now.